docs/plus/volume.md

```diff
@@ -36,3 +36,39 @@ const redis = pod.addContainer({
 // mount to the redis container.
 redis.mount('/var/lib/redis', data);
 ```
+
+## Using PersistentVolumeClaim Templates with StatefulSets
+
+When working with StatefulSets, you can use PersistentVolumeClaim templates to create stable storage for each pod in the StatefulSet. This allows each pod to have its own storage that persists even if the pod is rescheduled to a different node.
+
+```typescript
+import * as kplus from 'cdk8s-plus-32';
+import { Size } from 'cdk8s';
+
+// Create a StatefulSet with a PVC template
+const statefulSet = new kplus.StatefulSet(this, 'StatefulSet', {
+  containers: [{
+    image: 'nginx',
+    volumeMounts: [{
+      path: '/data',
+      name: Volume.fromName(chart, "pvc-template", "data-volume"),
```

Comment:
To provide a more idiomatic example, extract this to a constant:

```ts
const volume = Volume.fromName(chart, "pvc-template", "data-volume");
```

Then use `volume.name` below in the template configuration.